daml-lf/language/daml-lf.bzl

```diff
@@ -59,9 +59,12 @@ PROTO_LF_VERSIONS = LF_VERSIONS
 # The subset of LF versions accepted by //daml-lf/encoder
 ENCODER_LF_VERSIONS = ["1.dev" if ver == "dev" else ver for ver in LF_VERSIONS]
 
+# We support older LF versions using an older compiler binary
+LEGACY_COMPILER_LF_VERSIONS = ENCODER_LF_VERSIONS
```

Comment:
This also contains `1.14` and `1.dev`, should it?

Maybe we should hardcode the list of versions supported by the legacy compiler, since it presumably isn't going to keep up with new LF versions.